java/core/src/main/java/sleeper/core/properties/instance/IngestProperty.java

```diff
@@ -44,9 +44,15 @@ public interface IngestProperty {
     UserDefinedInstanceProperty INGEST_KEEP_ALIVE_PERIOD_IN_SECONDS = Index.propertyBuilder("sleeper.ingest.keepalive.period.seconds")
             .description("The frequency, in seconds, with which change message visibility requests are sent to extend the " +
                     "visibility of messages on the ingest queue so that they are not processed by other processes.\n" +
-                    "This should be less than the value of sleeper.queue.visibility.timeout.seconds.")
+                    "This should be less than the value of sleeper.ingest.queue.visibility.seconds.")
             .defaultValue("300")
             .propertyGroup(InstancePropertyGroup.INGEST).build();
+    UserDefinedInstanceProperty INGEST_QUEUE_VISIBILITY_TIMEOUT_IN_SECONDS = Index.propertyBuilder("sleeper.ingest.queue.visibility.timeout.seconds")
+            .description("The visibility timeout in seconds used for ingest queues. " +
+                    "This should be greater than sleeper.ingest.keepalive.period.seconds.")
+            .defaultValue("900")
+            .validationPredicate(SleeperPropertyValueUtils::isValidLambdaTimeout)
```

Comment:
The validation predicate seems incorrect, it's using a predicate for lambda timeout rather than a queue visibility timeout.